tests/RegistryPageFunctionalTest.php

```diff
@@ -82,6 +82,37 @@ public function testFilteredByRelationSearchResults()
         $this->assertEquals('Sherson', (string) $cells[1]->a[0]);
     }
 
+    /**
+     * Check that RegistryPageController can filter for ExactMatches (ID) for relationships.
+     *
+     * @throws \Exception
```

Comment:
Tests shouldn't throw exceptions. Are you expecting one here instead?

Reply:
tests do throw exceptions. $this->assert does exactly that doesn't it. I think it was PHP storm adding things for me. I can remove it if it makes you happier.

Reply:
Yeah if you don't mind, we don't usually annotate phpunit's internal exceptions